common/git-hooks/pre-commit

```diff
@@ -3,16 +3,15 @@
 # exit with non-zero status after issuing an appropriate message if
 # it wants to stop the commit.
 
-COMMAND=node scripts/node_modules/lint-staged/index.js
-
 echo --------------------------------------------
 echo Starting Git hook: pre-commit
 
-if [ -f $COMMAND ]; then
-  echo Invoking $COMMAND
-  $COMMAND
-else
-  echo Command not installed: $COMMAND
+echo Invoking node scripts/node_modules/lint-staged/index.js
```

Comment:
any reason to delete the COMMAND variable?

```
COMMAND=node scripts/node_modules/lint-staged/index.js
echo Invoking $COMMAND
$COMMAND
```

Reply:
When I was setting the variable, it was actually just running the command. I suppose one option is to store scripts/node_modules/lint-staged/index.js as a variable?

Reply:
wait... how did it work before??

Reply:
I guess it was just invoking the command during the variable assignment :O And then not doing anything for the code below that

Reply:
i guess just keep on keeping on with your original then.